tests/Phinx/Db/Adapter/MysqlAdapterTest.php

```diff
@@ -183,12 +188,12 @@ public function testCreateTableWithComment()
         $this->assertFalse($this->adapter->hasColumn('ntable', 'address'));
 
         $rows = $this->adapter->fetchAll(sprintf(
-            "SELECT table_comment FROM INFORMATION_SCHEMA.TABLES WHERE table_schema='%s' AND table_name='ntable'",
+            "SELECT TABLE_COMMENT FROM INFORMATION_SCHEMA.TABLES WHERE table_schema='%s' AND table_name='ntable'",
```

Comment:
Why doesn't this update all column names?

Reply:
The columns in the where clause are case insensitive, so didn't think to update them. Done now.